docs/index.rst

```diff
@@ -1,32 +1,32 @@
-.. Universal Ctag documentation master file
+.. Universal Ctags documentation master file
 
 #####################################################################
-universal-ctags hacking guide
+Universal-ctags Hacking Guide
 #####################################################################
 
 :Version: Draft
 :Authors: universal-ctags developers
 :Web Page: https://ctags.io/
 
-universal-ctags has the objective of continuing the
-development from what existed in the `Sourceforge`_ area. GitHub
-exuberant-ctags repository was started by Reza Jelveh
-<reza.jelveh@gmail.com> and was later moved to the `universal-ctags
-organization`_.
+`Universal-ctags`_ was created to continue the development of Darren
+Hiebert's `Exuberant-ctags`_ after activity on that project
+unfortunately stalled.
 
-The goal of the project is preparing and maintaining common/unified
-space where people interested in making ctags better can work
-together.
+Reza Jelveh <reza.jelveh@gmail.com> initially created a personal fork
+on GitHub and as interest and participation grew it was decided to
+move development to a dedicated GitHub organization.
```

Comment:
If you want to include this bit of history it might be worth expanding upon it with your role.